### PR TITLE
[test]Fix test error due to CRT dependency

### PR DESCRIPTION
**Pull request by earnol**
Remove CRT dependency in headers stdint.h, stdio.h inside test
clang/test/CodeGen/bit-int-ubsan.c

### CODE REVIEW

File: clang/test/CodeGen/bit-int-ubsan.c [hunk 1]
```diff
@@ -3,9 +3,7 @@
 
 // The runtime test checking the _BitInt ubsan feature is located in compiler-rt/test/ubsan/TestCases/Integer/bit-int.c
 
-#include <stdint.h>
-#include <stdio.h>
-
+typedef unsigned int uint32_t;
```
Comment (AaronBallman) [inline clang/test/CodeGen/bit-int-ubsan.c:6]: Can we fix the RUN line while we're at it? It's using `%clang` which runs the Clang driver, it should be using `%clang_cc1` so it only executes the frontend.

**Review by AaronBallman — COMMENTED** [review]
Changes generally LG, but I did have a question about another test improvement.

File: clang/test/CodeGen/bit-int-ubsan.c (hunk 1 above)
Comment (earnol) [reply clang/test/CodeGen/bit-int-ubsan.c:6]: Makes sense. Will update.

**Review by AaronBallman — APPROVED** [review]
LGTM once CI comes back green, thank you!

**Pull request merged**